src/getCompressionType.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const zlib = require('zlib');
+
+const BROTLI = 'brotli';
+const DEFLATE = 'deflate';
+const GZIP = 'gzip';
+
+const getCompressionType = (compressionType) => {
+  switch (compressionType) {
```

Comment:
It is recommended to handle the case when the 'compressionType' parameter does not match any of the predefined types. Enter an error message or exception for unsupported compression types to enhance error handling.